packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -246,6 +247,9 @@ class InspectorController extends DisposableController
   /// inspector tree load timing.
   bool firstInspectorTreeLoadCompleted = false;
 
+  final HoverCardTooltipController hoverCardTooltipController =
```

Comment:
this shouldn't be on the inspector controller. Lets crease a separate top level controller with a provider instead. could be bugs on the inspector page as sometimes it is going through the debugger controller and sometimes the inspector one.